### PR TITLE
feat(statics): add cosmos side chains config

### PR DESCRIPTION
**Pull request by mukeshsp**
Ticket: BG-78915

### CODE REVIEW

File: modules/statics/src/networks.ts [hunk 1]
```diff
@@ -745,18 +796,21 @@ export const Networks = {
...
     fiat: Object.freeze(new Fiat()),
+    hash: Object.freeze(new Hash()),
     hedera: Object.freeze(new Hedera()),
     litecoin: Object.freeze(new Litecoin()),
     polygon: Object.freeze(new Polygon()),
     ofc: Object.freeze(new Ofc()),
     osmo: Object.freeze(new Osmo()),
     rbtc: Object.freeze(new Rbtc()),
     stellar: Object.freeze(new Stellar()),
+    sei: Object.freeze(new SEI()),
```
Comment (DinshawKothari) [inline modules/statics/src/networks.ts:807]: Why SEI and not Sei() here ??

File: modules/statics/src/base.ts [hunk 2]
```diff
@@ -1258,6 +1266,10 @@ export enum BaseUnit {
   OFC = 'ofcCoin',
   OSMO = 'uosmo',
   FIAT = 'fiatCoin',
+  TIA = 'utia',
+  HASH = 'nHASH',
```
Comment (DinshawKothari) [inline modules/statics/src/base.ts:1270]: make it nhash

File: modules/statics/src/networks.ts (hunk 1 above)
Comment (mukeshsp) [reply modules/statics/src/networks.ts]: updated !!

File: modules/statics/src/base.ts (hunk 2 above)
Comment (mukeshsp) [reply modules/statics/src/base.ts]: updated !

**Review by DinshawKothari — APPROVED** [review]

**Pull request merged**